.yamato/build_linux_x64_debug.yml

```diff
@@ -11,7 +11,10 @@ variables:
   ARTIFACT_FILENAME: {{globals.artifact_base_name}}-linux-x64.7z
 
 commands:
-  - ./.yamato/scripts/build_linux.sh Debug
+  - mkdir artifacts
+  - curl https://public-stevedore.unity3d.com/r/public/7za-linux-x64/e6c75fb7ffda_e6a295cdcae3f74d315361883cf53f75141be2e739c020035f414a449d4876af.zip --output artifacts/7za-linux-x64.zip
```

Comment:
Now that I see how the zip downloading looks, maybe this would be better to move into C#.  Maybe put download & extraction behind `if(RunningOnYamato)`.  Or make it run locally but add a crude exists check caching scheme.

My concern is that sprinkling the steve hashes around is not ideal.

Reply:
I'll have to take a look but the hashes are usually there in some form, just typically in the manifest file instead of being referenced directly like this.

Reply:
yes, I just meant that the has could be consolidated to one place.

Reply:
Poor man's is fine

Reply:
Poor man's version has been implemented